core/frontend/services/redirects/settings.js

```diff
@@ -142,8 +142,14 @@ const setFromFilePath = (filePath, ext) => {
         });
 };
 
+const defaultJsonFileContent = '[]';
+
 const get = () => {
     return getCurrentRedirectsFilePath().then((filePath) => {
+        if (filePath === null) {
```

Comment:
I'm torn between handling this situation hear vs leaving it to [readRedirectsFile](https://github.com/TryGhost/Ghost/pull/12187/commits/b6252efe576c0e766f7987231f393e04ef901853#diff-48644be82a9b957e5e627bf7b0f2f73cdb1d63851ffad68c7c178c5886495bb8R16) to return the `[]` as it would in case the file does not exist. What are your thoughts?

Reply:
I checked the code and decided not to change things because `readRedirectsFile` is also used at `setFromFilePath`. If we add it to `readRedirectsFile`, it causes error because `JSON.parse([])` throws an error. 

I also thought about returning string `'[]'` inside `readRedirectsFile`. But it is also not a good idea because it's a bit unintuitive. It's not the result of reading a file.